src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -138,7 +138,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
      * > {@link Facet.options.dropdownHeaderLabel} and {@link Tab.options.dropdownHeaderLabel}).
      * >
      * > Furthermore, it is possible to specify the pixel threshold at which Facet components will go in responsive
-     * > mode (see {@link Facet.options.responsiveBreakpoint}.
+     * > mode (see {@link Facet.options.responsiveBreakpoint}).
```

Comment:
Thanks! Should I also deprecate [Facet.options.responsiveBreakpoint](https://coveo.github.io/search-ui/components/facet.html#options.responsivebreakpoint) then?